src/views/Dashboard.tsx

```diff
@@ -1,7 +1,15 @@
 import React from 'react';
+import DashboardHeader from '../components/DashboardHeader';
 
 const Dashboard: React.FunctionComponent = () => {
-    return <h1>Dashboard</h1>;
+    return (
+        <React.Fragment>
+            <h1>
+                Dashboard
+                <DashboardHeader />
```

Comment:
DashboardHeader should be outside of the h1

Reply:
Done.